src/core-styling/_types.scss

```
@@ -0,0 +1,7 @@
+@use "settings/font";
+@use "~sass-rem/rem" with (
+  $rem-baseline: font.$size,
+  $rem-fallback: false
+);
+
+@forward "~sass-rem/rem";
```

```suggestion
@forward "~sass-rem/rem" show rem;
```

(Not tested; does this forward both the mixin and the function?)

---

Don't like the ambiguity there. `@forward` removed in the refactor, see https://github.com/libero/storybook/pull/21/files#diff-cfeef31a60b46e4584db50a2f02af416